modules/speller/default/vector_hash-t.hpp

```diff
@@ -183,7 +183,8 @@ namespace aspeller {
   template<class Parms>
   void VectorHashTable<Parms>::recalc_size() {
     size_ = 0;
-    for (iterator i = begin(); i != this->e; ++i, ++this->_size);
+    iterator e = end();
+    for (iterator i = begin(); i != e; ++i, ++this->size_);
```

Comment:
Lets make this.
```
for (iterator i = begin(), e = end(); i != e; ++i, ++size_);
```

Please test with GCC 15.

Reply:
Tested on `gcc-15`. Builds successfully. Updated the PR.